### PR TITLE
Phalanx: add example of View of Views for no-UVM

### PR DESCRIPTION
**Pull request by rppawlo**
Add example code for view of views without UVM.

### CODE REVIEW

File: packages/phalanx/test/Kokkos/tKokkosViewOfViews.cpp [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    Kokkos::deep_copy(v_dev,v_host);
+
+    auto policy = Kokkos::MDRangePolicy<Kokkos::Rank<3>>({0,0,0},{num_cells,num_pts,num_equations});
+    Kokkos::parallel_for("view of view test",policy,KOKKOS_LAMBDA (const int cell,const int pt, const int eq) {
+      v_dev(2)(cell,pt,eq) = v_dev(0)(cell,pt,eq) + v_dev(1)(cell,pt,eq);
+    });
+
+    auto c_host = Kokkos::create_mirror_view_and_copy(Kokkos::HostSpace(),c);
+    Kokkos::deep_copy(c_host,c);
+
```
Comment (bathmatt) [inline packages/phalanx/test/Kokkos/tKokkosViewOfViews.cpp:42]: already copied above

File: packages/phalanx/test/Kokkos/tKokkosViewOfViews.cpp [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+TEUCHOS_UNIT_TEST(ViewOfViews,double) {
+
+  const int num_cells = 10;
+  const int num_pts = 8;
+  const int num_equations = 32;
+
+  Kokkos::View<double***,mem_t> a("a",num_cells,num_pts,num_equations);
+  Kokkos::View<double***,mem_t> b("b",num_cells,num_pts,num_equations);
+  Kokkos::View<double***,mem_t> c("c",num_cells,num_pts,num_equations);
+
```
Comment (bathmatt) [inline packages/phalanx/test/Kokkos/tKokkosViewOfViews.cpp:17]: Do you want to use PHX::View here??

**Comment by bathmatt** [comment]
@rppawlo i approve this but have some minor comments, i disabled automerge if you wanted to address or ignore,

**Review by bathmatt — APPROVED** [review]
didn't want automerge on so you can merge by hand or wait for @crtrott

File: packages/phalanx/test/Kokkos/tKokkosViewOfViews.cpp (hunk 1 above)
Comment (bathmatt) [reply packages/phalanx/test/Kokkos/tKokkosViewOfViews.cpp:42]: isn't line 40 a copy of c->c_host??

File: packages/phalanx/test/Kokkos/tKokkosViewOfViews.cpp (hunk 2 above)
Comment (rppawlo) [reply packages/phalanx/test/Kokkos/tKokkosViewOfViews.cpp:17]: This directory is to verify that kokkos code works for phalanx use cases before I screw things up with phalanx code. For now, keeping it phalanx free. I'll add another test with phalanx data structures soon.

File: packages/phalanx/test/Kokkos/tKokkosViewOfViews.cpp (hunk 1 above)
Comment (rppawlo) [reply packages/phalanx/test/Kokkos/tKokkosViewOfViews.cpp:42]: oops. I thought you were pointing to the other deep copy a few lines above. I'll fix that. Thanks!

File: packages/phalanx/test/Kokkos/tKokkosViewOfViews.cpp (hunk 2 above)
Comment (bathmatt) [reply packages/phalanx/test/Kokkos/tKokkosViewOfViews.cpp:17]: You don't need one, but just cheking,

**Comment by rppawlo** [comment]
@bathmatt - can you reapprove? Just pushed the fix.

**Review by bathmatt — DISMISSED** [review]

**Comment by rppawlo** [comment]
@trilinos/framework - can you tell me why this PR has not merged? It was reviewed and approved. The tests have passed. All conversations are marked resolved. It is stuck on the "Premerge Inspection" with a status of Pending since yesterday afternoon.

**Review by bathmatt — APPROVED** [review]

**Review by bathmatt — APPROVED** [review]

**Comment by bathmatt** [comment]
I tried everything to approve this, and get it to merge, looks like it is stuck somewhere and I'm not sure how to resolve.  

@trilinos/framework any ideas?

**Pull request merged**